app/src/main/java/io/branch/branchlinksimulator/MainActivity.kt

```diff
@@ -172,6 +193,72 @@ fun MainContent(navController: NavController) {
     customerEventAlias = savedAlias
     var aliasValue by remember { mutableStateOf(savedAlias) }
 
+    if (showAliasDialog) {
+        AlertDialog(
+            onDismissRequest = { showAliasDialog = false },
+            title = { Text("Enter Customer Event Alias") },
+            text = {
+                TextField(
+                    value = aliasValue,
+                    onValueChange = { aliasValue = it },
+                    label = { Text("Ex. mainAlias") },
+                )
+            },
+            confirmButton = {
+                TextButton(onClick = {
+                    sharedPreferences.edit().putString("customer_event_alias", aliasValue).apply()
+                    customerEventAlias = aliasValue
+                    Toast.makeText(context, "Set Customer Event Alias to $aliasValue", Toast.LENGTH_SHORT).show()
+                    showAliasDialog = false
+                }) {
+                    Text("Save")
+                }
+            }
+        )
+    }
+
+    if (showSessionIdDialog) {
+        AlertDialog(
```

Comment:
i pulled these alertdialogs out of the lazyColumn because they do not need to be there